LDAR_Sim/model_code/default_parameters/default_truck_parameters.py

```diff
@@ -0,0 +1,49 @@
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        default_truck_parameters
+# Purpose:     Default truck parameters
+#
+# Copyright (C) 2018-2020  Thomas Fox, Mozhou Gao, Thomas Barchyn, Chris Hugenholtz
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+
+default_truck_parameters = {
+    'version': '2.0',
+    'parameter_level': 'method',
+    'label': 'truck',
+    'module': 'truck',
```

Comment:
All the rest are "dummy" - should this be dummy too?

Reply:
@KeeganShaw-GIS  - all the 'dummy' modules will eventually be things like 'truck' etc. once testing of method PR is complete??

Reply:
Actually, they will be removed. The module does not provide anything anymore as its split up as a deployment type and a sensor type.

Reply:
Decision made:
- fix after testing method modules
- maybe split to two sub-dictionaries with 'deployment' and 'sensor'
- thought required on how this should be, so leaving this for a refit in coming week